src/cap-description.adoc

```diff
@@ -81,6 +81,19 @@ The byte-address of a memory location is encoded as MXLEN integer value.
 ^| 64      ^| {cap_rv64_addr_width}
 |==============================================================================
 
+[#section_cap_level,reftext="capability level"]
+==== Capability Level (CL)
+Capabilities can be assigned a level, which in conjunction with the <<sl_perm>> and <<ll_perm>> allows enforcing invariants on capability propagation.
+For example, this can be used to ensure that a callee cannot save a copy of the passed in argument or to avoid sharing of compartment-local data between compartments.
```

Comment:
By "save", do you mean write to memory?

Reply:
Yes, should probably make that explicit!